bundle/src/files.rs

```diff
@@ -147,6 +154,7 @@ impl FileSet {
             file_set_type: FileSetType::Junit,
             files,
             glob: glob_path,
+            test_runner_status,
```

Comment:
It looks like we only send `glob_with_status` so that we can immediately return it?

Reply:
Only in the case of extracting failures